temp.py

```diff
@@ -0,0 +1,8 @@
+def reset_files(filepath, file_type="txt"):
+    with open(filepath, "w") as output_file:
```

Comment:
Add error handling for file operations to catch and log exceptions such as `FileNotFoundError` and `PermissionError`. [important]